src/components/Identicon/__snapshots__/StatusIcon.test.tsx.snap

```diff
@@ -112,9 +112,9 @@ exports[`StatusIcon with account renders children in correct order 1`] = `
     class="c1"
   >
     <img
-      alt="Install MetaMask icon"
```

Comment:
this seems wrong

Reply:
tested manually and it looks to be working correctly.

Reply:
zach tested and says it's okay